src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -21,10 +21,10 @@ const TooltipTest = (props: Partial<TooltipProps>) => {
       text='Tooltip text'
       offset={2}
       placement='bottom-start'
-      container={document.getElementById('portal-root')}
+      container={document.getElementById('portal-root') ?? undefined}
```

Comment:
This doesn't make any sense 😅 

```suggestion
      container={document.getElementById('portal-root')!}
```
Doesn't work?